### DISCUSSION TITLE
Slow down scrolling on hitting Upper and Lower Page End

### DISCUSSION THREAD

**Discussion by blitzve0** [idiotWu/smooth-scrollbar#398] [Ideas]
Currently when we keep scrolling and reach the end of the page or top of the page, the page abruptly stops on hitting the end. If the scroller would slow down and smooth out when hitting the edges the experience would be much more impressive

**Comment by dolphin-wood** [comment]
We already got a solution for this! https://github.com/idiotWu/smooth-scrollbar/issues/198#issuecomment-496264237

P.S. IIRC, this was the default behavior before smooth-scrollbar@7

**Reply by blitzve0** [reply]
Yes I'm already using this solution, Maybe if this is the default behaviour that will be great for future users

**Reply by dolphin-wood** [reply]
Planned in official plugins :)
